### PR TITLE
Topic algorithms - 42370057 algorithms 

### PR DESCRIPTION
**Pull request by CameronGordon0**
Ported intensity_range and rescale_intensity from the scikit exposure module.  Files introduced in the 'PatternFlow/image directory, on the topic-algorithms branch. Driver file and README are included.

### CODE REVIEW

File: image/42370057_README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (shakes76) [inline image/42370057_README.md]: Please move files into own algorithm directory, including readme. Readme missing dependencies list

File: image/algorithm42370057.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
+
+"""
```
Comment (shakes76) [inline image/algorithm42370057.py]: More information header block would be good. Block would include authorship, license, copyright, dates and description of what the driver does.

File: image/algorithm42370057.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    Note: Tensorflow 2.0 is required to be used for this module. 
+
+    Parameters: 
+    image: array 
+    in_range, out_range : str or 2-tuple, optional
+        Min and max intensity values of input and output image. 
+        May be dtype or specified ranges. 
+    """
+
+    tf_dtype = tf.as_dtype(image.dtype)
```
Comment (shakes76) [inline image/algorithm42370057.py]: Good to add inline comments even if original code didn't have them

File: image/driver42370057.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    Author: Cameron Gordon, 42370057
+    Performs a test using the scipy raccoon face as an example image.
+    Then calls intensity_test, which is then plotted. 
+    """ 
+    fig = plt.figure(figsize=(10,10))
+    ax = plt.subplot(1,2,1)
+    face = misc.face()
+    plt.imshow(face)
+    
+    image = face # example image
```
Comment (shakes76) [inline image/driver42370057.py]: Inline comments would be helpful

**Comment by shakes76** [comment]
## TF Usage
Acceptable
## Good Practice (Design/Commenting)
Good design and commenting
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage 
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time) Algorithm not in directory
Feedback required, move algorithm files into its own directory, remove pyc files.
Request Description  minimal. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.

**Comment by iciac** [comment]
Incorporated feedback from code review. Moved algorithm files to their own directory 42370057, including readme. Included licence, author, and header information. Included inline comments. Readme includes dependency of Tensorflow 2.0. Removed pyc files from directory.

**Pull request merged**